src/Puppeteer.re

```diff
@@ -116,3 +116,32 @@ external launch : (~options: launchOptions=?, unit) => Js.Promise.t(Browser.t) =
 [@bs.val]
 [@bs.module "puppeteer"]
 external defaultArgs : unit => array(string) = "";
+
+type browserFetcherOptions = {
+  .
+  "host": Js.undefined(string),
+  "path": Js.undefined(string),
+  "platform": Js.undefined(string),
+};
+
+[@bs.obj]
+external makeBrowserFetcherOption :
```

Comment:
To be consistent this should be `makeBrowserFetcherOptions`, plural.

Reply:
Fixed in commit 784ba5c